packages/ffe-form-react/src/PhoneNumberField.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {string, bool, func} from 'prop-types';
+
+import i18n from './i18n/i18n';
+
+export default function PhoneNumberField({
+                               phone,
+                               countryCode,
+                               onChange,
+                               onBlur,
+                               locale,
+                               disabled,
+                               showValidationError,
+                               className
+                           }) {
+
+    const supportedLocales = ['nb', 'nn', 'en'];
+    const text = i18n[supportedLocales.indexOf(locale) !== -1 ? locale : 'nb'];
+
+    function onPhoneChange(e) {
+        onChange(countryCode, e.target.value);
+    }
+
+    function onCountryCodeChange(e) {
+        onChange(e.target.value, phone);
+    }
+
+    function handleBlur(e) {
+        if (e.target.id !== 'country-code' || e.target.value.length === 0) {
+            onBlur(e);
+        }
+    }
+
+    return (
+        <div className={`ffe-input-group ${className ? className : 'phone-number-field'}`}>
+            <div className="phone-number-field">
+                <div className="phone-number-field__country-code">
+                    <label className="ffe-form-label" htmlFor="country-code">{text.COUNTRY_CODE}</label>
+                    <div className="phone-number-field__input-group">
+                        <span className="phone-number-field__plus">+</span>
+                        <input
+                            id="country-code"
+                            maxLength="3"
+                            className="ffe-input-field phone-number-field__country-code-input"
+                            type="tel"
+                            disabled={disabled}
+                            value={countryCode}
+                            aria-invalid={showValidationError}
+                            onChange={onCountryCodeChange}
+                            onBlur={handleBlur}
+                        />
+                    </div>
+                </div>
+                <div className="phone-number-field__number">
+                    <label className="ffe-form-label" htmlFor="phone-input">{text.PHONE_NUMBER}</label>
+                    <input
+                        id="phone-input"
```

Comment:
specifying set IDs like this can be a potential problem. If you ever want to mount two of these on the same page, the labels won|'t know which field to focus once clicked!

This is an issue we've encountered before in other components. We typically create a unique ID for the component, like this:

```js
import uuid from 'uuid';

class PhoneNumber extends Component {
  id = uuid.v4();
  ...
  render() {
    return (...
      <label htmlFor={`phone-input-${id}`} />
      <input id={`phone-input-${id}`} />
      ...
    );
  }
}
```